src/epcpm/tests/test_sunspectoxlsx.py

```diff
@@ -67,7 +68,7 @@ def test_x():
         wrapped=attrs_model.root,
         parameter_uuid_finder=attrs_model.node_from_uuid,
         parameter_model=project.models.parameters,
-        column_filter=epcpm.sunspectoxlsx.attr_fill(epcpm.sunspectoxlsx.Fields, True),
+        column_filter=epcpm.pm_helper.attr_fill(epcpm.sunspectoxlsx.Fields, True),
```

Comment:
Is this change because we do not use excel (xlsx) ? .. did we change the process for this

Reply:
ok thanks for the clarification